rkt.spec

```diff
@@ -97,10 +120,24 @@ install -dp -m 700 %{buildroot}%{_sharedstatedir}/%{name}/containers
 %{_unitdir}/%{name}-metadata.socket
 %{_unitdir}/%{name}-metadata.service
 
-%{_sharedstatedir}/%{name}/cas
-%{_sharedstatedir}/%{name}/cas/db
-%{_sharedstatedir}/%{name}/tmp
-%{_sharedstatedir}/%{name}/containers
+%attr(2770, root, rkt) %{_sharedstatedir}/%{name}
```

Comment:
Is that supposed to be a simple `rkt install` or are there any other arguments/options following it? That'd be amazing. I'm guessing that step would handle setting directory perms in /var/lib/rkt, so I can only list /var/lib/rkt in the rpm files list, and not worry about setting perms in the spec.